apps/src/constants.js

```diff
@@ -57,6 +58,7 @@ exports.TestResults = {
 
   // Codes for unvalidated levels.
   UNSUBMITTED_ATTEMPT: -50,           // Progress was saved without submitting for review, or was unsubmitted.
+  UNVALIDATED_SUBLEVEL: -49,          // Either a member of a LevelGroup or a contained level, for which the result has not been validated
```

Comment:
Nope. I guess probably I should either have it either be -51, or move it in front of UNSUBMITTED_ATTEMPT (since our other negative ordered numbers are sorted that way in the file).